elements/rh-footer/RhFooter.ts

```diff
@@ -42,11 +42,12 @@ function isHeaderTagName(tagName: string) {
  * @slot    global - must contain `<rh-footer-universal>` {@deprecated - use `universal`}
  * @slot    universal - must contain `<rh-footer-universal>`
  *
+ * @cssprop --rhds-section-side-gap - {@default 16px, 32px, 64px}
```

Comment:
I've got a few concerns about this:

- Since this is global, it should go in tokens.
- This innovates `--rhds`, which we don't have anywhere else
- I'm not sure we should add this now during RC, when we'll certainly have to support it for a long time and in many places.

As a work-alike, can we just document this using spacing tokens?

Reply:
I share your concerns on all of the above.  I'll remove it in favor of just documentation for now.